help/start.txt

```diff
@@ -26,7 +26,7 @@ Usage: fastify start [opts] <file>
 
   -w, --watch
   [env: FASTIFY_WATCH]
-      Watch files that the file is change, the process will auto reload
+      Watch process.cwd()'s directory that the directory is change, the process will auto reload
```

Comment:
Watch `process.cwd()` directory for changes, recursively; when that happens, the process will auto reload.

Reply:
Thank you very much.